packages/flutter_tools/test/data/intellij/plugins/META-INF/plugin.xml

```diff
@@ -0,0 +1,93 @@
+<idea-plugin version="2">
```

Comment:
The code reads the META-INF/plugin.xml file from inside the jar files, so I'm wondering why this file needs to be checked in?